bit/find_missing_number.py

```diff
@@ -0,0 +1,19 @@
+def find_missing_number(nums):
+    """Returns the missing number from a sorted list of integers
+    in O(n) time and space.
+
```

Comment:
What is the nature of the sorted list of integers here? Can difference between consecutive integers be more than 1?